build.gradle

```diff
@@ -65,6 +65,6 @@ apply from: 'cppSettings.gradle'
 apply from: 'ni-libraries/ni-libraries.gradle'
 
 task wrapper(type: Wrapper) {
-    gradleVersion = '3.3'
-    distributionType = Wrapper.DistributionType.ALL
+    gradleVersion = '4.0.1'
```

Comment:
We switched to the all distribution so that we could get intelligent completion suggestions in the ide, as suggested by @JLLeitschuh.